tests/User/UserTest.php

```diff
@@ -199,15 +199,17 @@ public function testUpdateQuotaAllProvided() {
 
 		$connection->expects($this->at(0))
 			->method('__get')
-			->with($this->equalTo('ldapQuotaDefault'))
-			->will($this->returnValue('23 GB'));
+			->with($this->equalTo('ldapQuotaAttribute'))
+			->will($this->returnValue('myquota'));
 
+		/* Having a quota defined, the ldapQuotaDefault won't be used
 		$connection->expects($this->at(1))
 			->method('__get')
-			->with($this->equalTo('ldapQuotaAttribute'))
-			->will($this->returnValue('myquota'));
+			->with($this->equalTo('ldapQuotaDefault'))
+			->will($this->returnValue('23 GB'));
```

Comment:
Yes, it was commented on purpose. Previously we were always fetching the default quota even though it wasn't being used in some scenarios. I think this way is a bit more clear that the default quota won't be fetched in this case
i'll check if I can do it properly.